checklist.md

```diff
@@ -0,0 +1,20 @@
+# Checklists
+
+## Suspending Someone
+
+1. Suspend on GitHub
+2. Suspend on Matrix with mjolnir
+3. Suspend on Discourse
+4. Remove inappropriate content
+5. Add entry to moderation-log.md
+  - Include the length of the suspension, if not indefinite
+  - Do __not__ include why the suspension was issued
+
+### Indefinite Suspensions
+
+1. Remove from Nixpkgs maintainers list and remove their maintainership of all packages
+2. Remove from NixOS org maintainer list and any other NixOS org permissions
+3. Notify major adjacent project administrators in case they also want to take action:
```

Comment:
Would it also make sense to close all the person's unmerged PRs? It has already been done in practice.

```suggestion
3. Close all open PRs by that person, noting the indefinite suspension
4. Notify major adjacent project administrators in case they also want to take action:
```